src/app/[locale]/page.tsx

```diff
@@ -1,5 +1,13 @@
-import styles from "./page.module.scss";
+import HomePage from "../../components/pages/Home/Home";
+import { getTopLatestExhibits } from "@/lib/exhibit";
+import { ITopLatestExhibit } from "@/interfaces/ITopLatestExhibit";
 
-export default function Home() {
-  return <div>Coming soon</div>;
+const topLatestExhibitsLimit = 10;
+
+export default async function Home() {
+  const topLatestExhibits = (await getTopLatestExhibits(
+    topLatestExhibitsLimit,
+  )) as ITopLatestExhibit[];
```

Comment:
Not sure that need type assertion here... I suppose type should be passed from appollo client

Reply:
I am afraid it does not work here, TS raises types error and it is mandatory to perform manual casting. I can try to put it on `lib` error to prevent extra casts in uses cases